talk/morelanguage/constexpr.tex

```diff
@@ -148,3 +142,27 @@
     static_assert(km_y == 1093, "expected km == 1093 yards!");
   \end{cppcode*}
 \end{frame}
+
+\begin{frame}[fragile]
```

Comment:
All good on this slide. Since the four cases are also here, it might be OK to only have two examples in the previous diff.